### DISCUSSION TITLE
How can I relocate the search bar in Nextra to different areas of the page, changing its position while ensuring it continues to function correctly?

### DISCUSSION THREAD

**Discussion by DanielgTeixeira1** [shuding/nextra#2587] [Q&A]
How can I relocate the search bar in Nextra to different areas of the page, changing its position while ensuring it continues to function correctly?

**Comment by jellli** [comment]
Are you using the doc theme right now?
You can custom your own search component with [this](https://nextra.site/docs/docs-theme/theme-configuration#search) config prop
